pinot-core/src/main/java/org/apache/pinot/core/util/GapfillUtils.java

```diff
@@ -189,15 +171,15 @@ public static GapfillType getGapfillType(QueryContext queryContext) {
     Preconditions.checkArgument(gapFillSelection != null && gapFillSelection.getFunction() != null,
         "Gapfill Expression should be function.");
     List<ExpressionContext> args = gapFillSelection.getFunction().getArguments();
-    Preconditions.checkArgument(args.size() > 5, "PreAggregateGapFill does not have correct number of arguments.");
+    Preconditions.checkArgument(args.size() > 5, "GapFill does not have correct number of arguments.");
```

Comment:
(minor) Treat `Gapfill` as one word, same for other exception messages
```suggestion
    Preconditions.checkArgument(args.size() > 5, "Gapfill does not have correct number of arguments.");
```